<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToolBarMeasureShowOptions</class>
 <widget class="QWidget" name="ToolBarMeasureShowOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>274</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ToolBarMeasureShowOptions</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>6</number>
   </property>
   <item row="1" column="1">
    <widget class="QCheckBox" name="checkBox_horizontal">
     <property name="text">
      <string>Horizontal</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QCheckBox" name="checkBox_total">
     <property name="text">
      <string>Distance</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QCheckBox" name="checkBox_vertical">
     <property name="text">
      <string>Vertical</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QCheckBox" name="checkBox_showValues">
     <property name="text">
      <string>Display Values</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>checkBox_showValues</tabstop>
  <tabstop>checkBox_total</tabstop>
  <tabstop>checkBox_horizontal</tabstop>
  <tabstop>checkBox_vertical</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
